components/storage/blockdevices/COMPONENT_DATAFLASH/TESTS/block_device/dataflash/main.cpp

```diff
@@ -0,0 +1,135 @@
+#include "mbed.h"
+#include "greentea-client/test_env.h"
+#include "unity.h"
+#include "utest.h"
+
+#include "DataFlashBlockDevice.h"
+#include <stdlib.h>
+
+using namespace utest::v1;
+
+#define TEST_PINS PTB22, PTB23, PTB21, PTB20
```

Comment:
@yossi2le : I meant this one.. apologies to comment in the wrong place previously..

Reply:
10x, This is being fixed right now and pushed on the next commit. I have missed that one.